docker-run.sh

```diff
@@ -1,7 +1,3 @@
 #!/bin/sh
 
-APP_ENV=$APP_ENV yarn build:preview && yarn start \
```

Comment:
these inline vars are no longer required because they exist inside of the `secrets.yaml` and `configmap.yaml` files?

Reply:
That's right, this is vestigial code from when we were thinking of using env vars from the command line, and not setting up the .env files as we do now. Leaving it is confusing.